src/js/edu-benefits/utils/validations.js

```diff
@@ -291,7 +291,9 @@ function isValidEmploymentHistoryPage(data) {
 }
 
 function isValidEducationPeriod(data) {
-  return isValidDateRange(data.dateRange.from, data.dateRange.to);
+  return (isBlankDateField(data.dateRange.from) || isValidDateField(data.dateRange.from))
```

Comment:
I'm a little confused about this fix. So we're allowing the from date and/or the to date to be totally blank? Is this required for the schema error fix?